### PR TITLE
[Data Integrity Flag] updated module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
New copy for module help text

#### Testing instructions

1. Review copy
2. see screenshot

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

Screenshot 2019-11-19 15 16 43

### CODE REVIEW

**Comment by christinerogers** [comment]
this sounds alllllot like the data team help / behavioural QC.  
note to myself - review the difference between the modules, to improve this help text

File: modules/data_integrity_flag/help/data_integrity_flag.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
-Within the results table (third table on the Data Integrity page), links under the "Instrument" column will redirect the user to a detailed summary of each field for that particular instrument along with links to behavioural feedback and to incomplete candidates. Users can also view the date the flag was submitted, the Flag Status, any comments, data cleaning feedback, and UserID. Note, Flag Status is based on the codes used in the dropdown of the second table, where 1=Ready for Review, 2= Review Completed, 3=Feedbacks Closed, and 4=Finalization.
+This module provides a direct way to view and update behavioural feedback at-a-glance.
+
+In the **Browse** tab, use the *Selection Filter* section to search for existing feedback. You can click any link in the *Instrument* column to view a detailed summary of each field for that instrument and visit.
+
+In the **Update** tab, you can add a new instrument Flag status. 
+You'll notice that  Flag Status is coded as: <br>
```
Comment (johnsaigle) [inline modules/data_integrity_flag/help/data_integrity_flag.md]: Is `<br>` the only way to create new lines? I don't think we want HTML tags to be present in these files.

Comment (knoll-alex) [reply modules/data_integrity_flag/help/data_integrity_flag.md]: I believe aside from adding a space between lines, the <br> tag is the only way to break up those lines with the dashes - and those would look silly with spaces between

**Comment by christinerogers** [comment]
@driusan almost ready to merge, needs one more review/approval

**Pull request merged**